.github/workflows/deploy.yml

```diff
@@ -15,9 +15,10 @@ jobs:
         with:
           python-version: '3'
       - name: Install OctoDNS
-        run: pip install 'octodns==0.9.10'
+        run: pip install 'octodns>=1.5.0' octodns-dnsimple octodns-cloudflare
```

Comment:
This isn't specifying the latest octoDNS version, why? The latest is 1.13.0. Additionally this doesn't use dependency pinning ([recommended by octodns](https://github.com/octodns/octodns-cloudflare?tab=readme-ov-file#requirementstxtsetuppy)) meaning there could be breaking changes in an upgrade later causing the workflow to fail.